test/extensions/transport_sockets/tls/test_data/certs.sh

```diff
@@ -143,6 +143,12 @@ generate_ecdsa_key selfsigned_ecdsa_p384 secp384r1
 generate_selfsigned_x509_cert selfsigned_ecdsa_p384
 rm -f selfsigned_ecdsa_p384_cert.cfg
 
+# Generate long_validity_cert.pem as a self-signed, with expiry that exceeds 32bit time_t.
```

Comment:
Nit: Could we set this far in the future expiration date in one of the existing certificates instead of adding a new one?

Reply:
I'm not sure which existing to use without losing the explicit intent of the test, eg snippets from other time specific certs :

```
# Generate expired_cert.pem as a self-signed, expired cert (will fail on macOS 10.13+ because of negative days value).
generate_x509_cert expired ca -365

# Generate expired_san_uri_cert.pem as a CA signed, expired cert (will fail on macOS 10.13+ because of negative days value).
generate_x509_cert expired_san_uri ca -365
```

Reply:
I think it's fine to keep the new cert here.